tiledb/cc/enum.cc

```diff
@@ -46,13 +46,17 @@ void init_enums(py::module &m) {
       DFENUM(ZSTD) DFENUM(LZ4) DFENUM(RLE) DFENUM(BZIP2) DFENUM(DOUBLE_DELTA)
           DFENUM(BIT_WIDTH_REDUCTION) DFENUM(BITSHUFFLE) DFENUM(BYTESHUFFLE)
               DFENUM(POSITIVE_DELTA) DFENUM(CHECKSUM_MD5)
-                  DFENUM(CHECKSUM_SHA256) DFENUM(DICTIONARY)
-                      DFENUM(SCALE_FLOAT);
+                  DFENUM(CHECKSUM_SHA256) DFENUM(DICTIONARY) DFENUM(SCALE_FLOAT)
```

Comment:
BTW we have refactored this `enum.cc` file quite a bit (mostly just syntax changes for readability) which you will see when you rebase against dev but just warning ahead of time.